src/test/java/org/opensearch/security/dlic/rest/api/AbstractRestApiUnitTest.java

```diff
@@ -213,12 +213,12 @@ protected String checkReadAccess(int status, String username, String password, S
 
 	}
 
-	protected String checkWriteAccess(int status, String username, String password, String indexName, String type,
+	protected String checkWriteAccess(int status, String username, String password, String indexName, String actionType,
```

Comment:
Fixing the naming here as OpenSearch 2.0+ supports actionType in the API path and dropped the support for type.